maps/away/magshield/magshield.dm

```diff
@@ -0,0 +1,136 @@
+	suffix = "magshield/magshield.dmm"
+	spawn_weight = 1
+	spawn_cost = 1
+
+/decl/submap_archetype/magshield
+	map = "magshield"
+	descriptor = "It's an orbital shield station."
+
+/obj/effect/overmap/visitable/sector/magshield
+	name = "orbital station"
+	desc = "Sensors detect an orbital station above an exoplanet. Signs of past magentic impulses are registred from it. Planet landing is impossible due to lower orbits being cluttered with chaotically moving metal chunks."
+	icon_state = "object"
+
+	initial_generic_waypoints = list(
+		"nav_magshield_1",
+		"nav_magshield_2",
+		"nav_magshield_3",
+		"nav_magshield_4",
+		"nav_magshield_5",
+		"nav_magshield_6"
+	)
+
+/obj/effect/shuttle_landmark/nav_magshield/nav1
+	name = "Orbital Station Navpoint #1"
+	landmark_tag = "nav_magshield_1"
+
+/obj/effect/shuttle_landmark/nav_magshield/nav2
+	name = "Orbital Station Navpoint #2"
+	landmark_tag = "nav_magshield_2"
+
+/obj/effect/shuttle_landmark/nav_magshield/nav3
+	name = "Orbital Station Navpoint #3"
+	landmark_tag = "nav_magshield_3"
+
+/obj/effect/shuttle_landmark/nav_magshield/nav4
+	name = "Orbital Station Navpoint #4"
+	landmark_tag = "nav_magshield_4"
+
+/obj/effect/shuttle_landmark/nav_magshield/nav5
+	name = "Orbital Station Navpoint #5"
+	landmark_tag = "nav_magshield_5"
+
+/obj/effect/shuttle_landmark/nav_magshield/nav6
+	name = "Orbital Station Navpoint #6"
+	landmark_tag = "nav_magshield_6"
+
+//mainly props now, the gimmick of an infrequent emp pulse was kind of interesting but hard to port and might punish synths too hard for exploring
+/obj/structure/magshield/maggen 
+	name = "magnetic field generator"
+	desc = "A large three-handed generator with rotating top. It is used to create high-power magnetic fields in hard vacuum. It's spinning and that seems to be all It's good for at the moment."
+	icon = 'maps/away/magshield/magshield_sprites.dmi'
+	icon_state = "maggen"
+	anchored = TRUE
+	density = TRUE
+	light_color = "#ffea61"
+
+/obj/structure/magshield/maggen/attack_hand(mob/user)
+	..()
+	to_chat(user, "<span class='notice'> You don't see how you could fix \the [src]. It would need some serious work.</span>")
+
+/obj/structure/magshield/rad_sensor
+	name = "radiation sensor"
+	desc = "Very sensitive vacuum radiation sensor. On top of the metal stand two modified Wilson Cloud Chambers filled with deuterium and tritium water."
+	icon = 'maps/away/magshield/magshield_sprites.dmi'
+	icon_state = "rad_sensor"
+	anchored = TRUE
+
+/obj/structure/magshield/nav_light
+	name = "navigation light"
+	desc = "Large and bright light regularly emitting green flashes."
+	icon = 'maps/away/magshield/magshield_sprites.dmi'
+	icon_state = "nav_light_green"
+	anchored = TRUE
+	density = TRUE
+
+/obj/structure/magshield/nav_light/New() //Would be cool if these could flash, but I couldn't figure out how to carry that over.
+	..()
+	set_light(1.2, 3, LIGHT_COLOR_GREEN)
+
```

Comment:
stray newline.

Reply:
Intended for cleaner formatting. Or at least, I usually prefer a space before the next line.